lib/chef/provider/package/debian.rb

```diff
@@ -0,0 +1,129 @@
+#
+# Author:: Kapil Chouhan (<kapil.chouhan@msystechnologies.com>)
+# Copyright:: Copyright 2008-2018, Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "chef/provider/package"
+
+class Chef
+  class Provider
+    class Package
+      module Debian
+        def self.included(base)
+          base.class_eval do
+            use_multipackage_api
+
+            action :reconfig do
+              if current_resource.version.nil?
+                logger.trace("#{new_resource} is NOT installed - nothing to do")
+                return
+              end
+
+              unless new_resource.response_file
+                logger.trace("#{new_resource} no response_file provided - nothing to do")
+                return
+              end
+
+              if preseed_file = get_preseed_file(new_resource.package_name, current_resource.version)
+                converge_by("reconfigure package #{new_resource.package_name}") do
+                  preseed_package(preseed_file)
+                  multipackage_api_adapter(new_resource.package_name, current_resource.version) do |name, version|
```

Comment:
```suggestion
                  multipackage_api_adapter(new_resource.package_name, current_resource.version) do |name, _version|
```
The revised `reconfig_package()` method now does not need the `version` parameter, so `version` here is not used. It would help future readers (and Rubocop) to know that `version` is _intentionally_ not used if you change `|name, version|` to `|name, _version|`

Reply:
we don't do that in the chef codebase though, and we went through that rule and intentionally and thoughtfully turned it off though.

Reply:
OK. Mine wasn't a strong suggestion.